### PR TITLE
103 highlight next meetup

### PR DESCRIPTION
**Pull request by kiramclean**

### CODE REVIEW

**Comment by kiramclean** [comment]
closes #127

**Comment by kiramclean** [comment]
closes #45

**Comment by sophiedeziel** [comment]
haha! Hound just bit you!
I'll review as soon as possible

**Comment by kiramclean** [comment]
haha I know the Hound is sassy!

**Comment by kiramclean** [comment]
Somehow I knew that wouldn't solve all the hound complaints haha.

**Comment by kiramclean** [comment]
@sophiedeziel Woohoo! No more single quotes or ambiguous regex's. I think this is finally ready for review. Sorry for the delay. :tada: :raised_hands:

**Comment by kiramclean** [comment]
:tada:

**Comment by sophiedeziel** [comment]
I just tried it and I have to say that you did a great job! It's more readable and I like what you did with the boxes and titles!

**Comment by kiramclean** [comment]
Thanks! Let me know what you think or if you have any ideas for changes.

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -1,20 +1,17 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
+@import "layouts/variables";
```
Comment (sophiedeziel) [inline app/assets/stylesheets/application.scss]: Good idea

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -2,18 +2,14 @@ class ApplicationController < ActionController::Base
...
   end
 
-  def get_next_meetup
-    @next_meetup = Meetup.next
-  end
-
-  def get_events
-    @events = Event.published
+  def load_sidebar
+    @news_items = NewsItem.published.to_a
```
Comment (sophiedeziel) [inline app/controllers/application_controller.rb]: Why `.to_a` ?

**Pull request merged**